subsys/bluetooth/mesh/net.c

```diff
@@ -36,6 +36,7 @@
 #include "cfg.h"
 #include "statistic.h"
 #include "sar_cfg_internal.h"
+#include "brg_cfg.h"
```

Comment:
Should this have "#if IS_ENABLED(CONFIG_BT_MESH_BRG_CFG_SRV)"? Coverity may complain otherwise (low risk warning).

Reply:
Seems we do not do it in any source code. Just line above for SAR model. Or one more above for statistic. The header section will look very cumbersome just to make happy Coverity :)

Reply:
According to coverity warnings, removing unwanted includes speeds up compilation. But I agree, if we don't do it for others (like proxy, friend, proxy_cli, etc.), then may be we should just leave it as is.